test/Conversion/VectorToAIEVec/test_mul_elem.mlir

```diff
@@ -37,3 +37,16 @@ func.func @test_mul_elem_i8(%a : vector<32xi8>,
   %3 = arith.muli %1, %2 : vector<32xi32>
   return %3 : vector<32xi32>
 }
+
+func.func @test_mul_elem_i8_i8(%a : vector<32xi8>,
+                         %b : vector<32xi8>) -> vector<32xi8> {
+  // CHECK:  %[[C0:.*]] = arith.constant 0 : i8
+  // CHECK:  %[[BCS:.*]] = aievec.broadcast_scalar %[[C0:.*]] : i8, vector<64xi8>
+  // CHECK:  %[[EXT:.*]] = aievec.ext %[[BCS:.*]] {index = 0 : i8} : vector<64xi8>, vector<32xi8>
+  // CHECK:  %[[CC1:.*]] = aievec.concat %[[A]], %[[EXT:.*]] : vector<32xi8>, vector<64xi8>
+  // CHECK:  %[[CC2:.*]] = aievec.concat %[[B]], %[[EXT:.*]] : vector<32xi8>, vector<64xi8>
+  // CHECK:  %[[ME:.*]] = aievec.mul_elem %[[CC1:.*]], %[[CC2:.*]] : vector<64xi8>, vector<64xi8>, vector<32xi32>
```

Comment:
Same with all of these. When you do `[[<name>:<regex>]]` you are capturing the `regex` in `<name>`, when you want to use the variable to establish a `use` from a `def`, you just use the name: `[[<name>]]`.

Reply:
Thanks for the comments, Javier. I will revise the code in my next change set.